### ISSUE TITLE
Delay on cc's 

### ISSUE THREAD

**Issue by Sunqtt** [RetroWoW/RetroWoW#199]
https://www.youtube.com/watch?v=ajosYuwNjSE

it needs to be fix asap, if a rogue opens another rogue, if that rogue spam blind he can blind the rogue that cheap shot him
a mage can get blind even if he sheeped his target
etc

**Comment by Maadguy** [comment]
Yeah to be honest CC delay has to be the most important issue in pvp right now, it breaks the whole thing hard
